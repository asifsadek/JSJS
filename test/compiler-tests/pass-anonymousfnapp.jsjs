val hw = (/\(s1, s2) => s1 ^ " " ^  s2;)("hello", "world");
print(hw);

val sum = (/\(x, y, z) => x + y + z;)(5, 10, 15);
print_num(sum);

print_bool((/\(x) => x >= 0;)(2));
