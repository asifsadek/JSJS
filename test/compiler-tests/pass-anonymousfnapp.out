hello world
30
true
